src/countdown-timer/types.ts

```diff
@@ -20,6 +20,8 @@ export interface CountdownTimerProps
     fixed?: boolean | undefined;
     /** Specifies the countdown timer (in seconds) */
     timer: number;
+    /** Specifies the timestamp at which the countdown ends (milliseconds since Jan 1, 1970) */
+    timestamp?: number;
```

Comment:
as `timer` and `timestamp` are mutually exclusive, `timer` should not be mandatory. but we need to ensure either of them is specified at any time. can we use discriminated union to ensure either `timer` or `timestamp` is specified at any given time?